### DISCUSSION TITLE
Prácticas Frontend

### DISCUSSION THREAD

**Discussion by Arrrleth** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#126] [Q&A] [answered]
Buenas noches compañeros. ¿Saben la fecha limite para subir estas primeras prácticas?

**Comment by Takzer94** [comment]
Mañana

**Comment by GloriaCoralCerecedo** [comment]
El MC comento que mañana pero que podías entregarlas tiempo después, por ejemplo el 28 o eso creo

**Comment by robertogarcer** [comment]
aun no se sube alguna liga o formulario verdad?

**Reply by Takzer94** [reply]
No, todavía no ha dado ninguna indicación

**Reply by OscarIslasH** [reply]
https://teams.microsoft.com/l/message/19:8a61b3259c3147259ba0197e9b103432@thread.tacv2/1645814720628?tenantId=4ae54b05-b77e-4224-aef1-8661422e0816&groupId=d38ceaac-3599-4f30-94ba-740ccb530cb0&parentMessageId=1645814720628&teamName=Misi%C3%B3n%20Frontend%20-%20JS&channelName=Semana%201%20-%20Despegue%20%F0%9F%9A%80&createdTime=1645814720628 

Ya lo pasaron es este

**Comment by romarpla** [comment, marked as answer]
[Link de publicación de Teams](https://teams.microsoft.com/l/message/19:8a61b3259c3147259ba0197e9b103432@thread.tacv2/1645814720628?tenantId=4ae54b05-b77e-4224-aef1-8661422e0816&groupId=d38ceaac-3599-4f30-94ba-740ccb530cb0&parentMessageId=1645814720628&teamName=Misi%C3%B3n%20Frontend%20-%20JS&channelName=Semana%201%20-%20Despegue%20%F0%9F%9A%80&createdTime=1645814720628)

Este es el link dentro del equipo de Teams para que puedan ver el link del Formulario
